charts/linkerd2-cni/values.yaml

```diff
@@ -90,16 +90,16 @@ extraInitContainers: []
 resources:
   cpu:
     # -- Maximum amount of CPU units that the cni container can use
-    limit: ""
+    limit: 
```

Comment:
I see these have changed from `""` to nil / null. Any particular reason for this? 🤔 My preference is to stick with `""` since Go panics on nil dereferences (and it provides us with better defaults). I'd have a look at what we do in the control plane though, we'll want to do the same thing that we do for the normal chart.

Reply:
I have solved it using yaml for marshal and unmarshal.